src/libutil/experimental-features.cc

```diff
@@ -37,20 +139,40 @@ const std::optional<ExperimentalFeature> parseExperimentalFeature(const std::str
         return std::nullopt;
 }
 
-std::string_view showExperimentalFeature(const ExperimentalFeature feature)
+std::string_view showExperimentalFeature(const ExperimentalFeature tag)
 {
-    const auto ret = get(stringifiedXpFeatures, feature);
-    assert(ret);
-    return *ret;
+    assert((size_t)tag < xpFeatureDetails.size());
+    return xpFeatureDetails[(size_t)tag].name;
+}
+
+std::string getExperimentalFeaturesList() {
+    std::string experimentalFeaturesList = R"(
+        Experimental Nix features to enable.
+        Current experimental features are the following:
```

Comment:
````suggestion
        Experimental features that can be enabled.
        
        Example:
        
        ```
        experimental-features = nix-command flakes
        
        Experimental features available:
````

@nixos/nix-team this would benefit from a link to or an inline summary of the experimental feature page once https://github.com/NixOS/nix/pull/5930 is merged, such that users know what to expect.